data/src/pixel.rs

```diff
@@ -477,6 +522,7 @@ impl fmt::Display for Formaton {
     }
 }
 
+// TODO: get rid of this macro
```

Comment:
The macro can be converted to a series of function `new_yuv8()`, `new_yuv_hb()`...